vnext/Microsoft.ReactNative/Modules/Animated/ValueAnimatedNode.cpp

```diff
@@ -96,12 +96,24 @@ void ValueAnimatedNode::AddActiveAnimation(int64_t animationTag) {
 
 void ValueAnimatedNode::RemoveActiveAnimation(int64_t animationTag) {
   m_activeAnimations.erase(animationTag);
+  m_stoppedAnimations.erase(animationTag);
   if (!m_activeAnimations.size()) {
     if (const auto manager = m_manager.lock()) {
+      // Dispose completed props animations
       for (const auto &props : m_dependentPropsNodes) {
         if (const auto propsNode = manager->GetPropsAnimatedNode(props))
           propsNode->DisposeCompletedAnimation(Tag());
       }
+
+      // Start any deferred animations, if the animation was stopped before
+      // this is run, we will get a null pointer for the animation driver. We
+      // also remove any deferred animation tags in StopAnimation.
+      for (const auto &deferredId : m_deferredAnimations) {
```

Comment:
Run this loop in a separate conditional that fires if there are no more stopped animations, rather than active animations.